packages/css/src/container.ts

```diff
@@ -0,0 +1,6 @@
+import { generateIdentifier } from './identifier';
+
+// createContainer is used for local scoping of CSS containers
+// For now it is mostly just an alias of generateIdentifier
```

Comment:
Nit: Why "for now"? Won't this always be the case?

Reply:
I guess I wasn't sure if there would ever be more functionality. Potentially the container could provide an API for querying itself.

For example:

```ts
const container = createContainer();

style({
  '@container': {
    [container('(min-width: 400px)')]: ...
  }
})

Reply:
Oh, that's interesting.